### PR TITLE
minor improvement to extractDataFromUnstructuredOutput.py

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
Some minor improvement to this script for extracting time steps:
- default to hdf5 and float as it drastically reduces file size.
- write in the current folder and not in the folder of the input file.
- add `last` option to write the last time step without knowing the index.
-fix for `-` sign in prefix

### CODE REVIEW

File: postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py [hunk 1]
```diff
@@ -11,8 +11,9 @@
 parser.add_argument("--add2prefix", help="string to append to prefix for new file", type=str, default="_resampled")
 parser.add_argument("--Data", nargs="+", metavar=("variable"), default=(""), help="Data to resample (example SRs)")
 parser.add_argument("--downsample", help="write one out of n output", type=int)
-parser.add_argument("--float", dest="float", default=False, action="store_true", help="convert to float")
-parser.add_argument("--hdf5", dest="hdf5", default=False, action="store_true", help="output as hdf5")
+parser.add_argument("--double", dest="double", default=False, action="store_true", help="do not convert to float")
+parser.add_argument("--posix", dest="posix", default=False, action="store_true", help="output as posix")
```
Comment (sebwolf-de) [inline postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py:15]: Maybe make these two arguments a string: like ```"--precision", type=str, choices=["float", "double"],  default="float"```

**Review by sebwolf-de — COMMENTED** [review]
Looks good to me, just one minor remark. But this is just for IMHO better usability.

File: postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py (hunk 1 above)
Comment (krenzland) [reply postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py:15]: Also the name posix is slightly confusing (has nothing to do with the actual file format). Not sure what (if anything) would be better. Maybe 'raw'?

Comment (Thomas-Ulrich) [reply postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py]: I used posix as it is the name used in xdmfWriterBackend. Do you think posix makes sense in xdmfWriterBackend?
I can use raw (at the end, probably this option will never be used anyway :-p (well I used it as some point when samoa was only reading bin files)).

Comment (Thomas-Ulrich) [reply postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py]: anyway, I've adressed all comments, thx for the review!

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**